Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -129,7 +129,18 @@ GameListItem::GameListItem(const std::string& _rFileName)
 		ini.GetIfExists("EmuState", "EmulationIssues", &m_issues);
 	}
 
-	if (!m_pImage.empty())
+	if (IsElfOrDol())
+	{
+		DiscIO::IVolume* pVolume = DiscIO::CreateVolumeFromFilename(_rFileName);
+
+		m_Bitmap.LoadFile(StrToWxStr(File::GetThemeDir(SConfig::GetInstance().theme_name)) + "filebanner.png", wxBITMAP_TYPE_PNG);
+		m_Country = DiscIO::IVolume::COUNTRY_UNKNOWN;
+		m_emu_state = 0;
+		m_disc_number = 0;
```

Comment:
You could move the initialization of `m_disc_number` and `m_Country` to the initializer list. `m_emu_state` is already in the initializer list and is thus unnecessary here.